docs/core/stories.rst

```diff
@@ -87,8 +126,8 @@ OR Statements
 -------------
 
 Another way to write shorter stories, or to handle multiple intents
-the same way, is to use an ``OR`` statement. For example if you ask
-the user to confirm something, and we want to treat the ``affirm``
+the same way, is to use an ``OR`` statement. For example, if you ask
+the user to confirm something, and you want to treat the ``affirm``
 and ``thankyou`` intents in the same way. The story below will be
 converted into two stories at training time. Just like checkpoints,
```

Comment:
Maybe this "just like checkpoints" sentence should go into the note below instead. it's confusing that we say "it's probably better to restructure your intents:" and then that's not what the story shows (the story shows "the story below will be converted..."